encrypted-media/scripts/check-hdcp-policy.js

```diff
@@ -0,0 +1,26 @@
+function runTest(config, qualifier)
+{
+  function checkHdcpPolicy(hdcpPolicy)
```

Comment:
This should be something like checkStatusForMinHdcpVersionPolicy(hdcpVersion)

Reply:
Done.